public/data/job-openings-data.json

```diff
@@ -231,6 +231,34 @@
           "experience": "-"
         }
       ]
+    },
+    {
+      "id": 5,
+      "companyName": "Ansi ByteCode LLP",
+      "location": "Ahmedabad",
+      "bio": " Ansi ByteCode LLP has developed an exceptional reputation within the Global IT industry. The company was launched in 2014 and since then, it has been a thriving center. We presently utilize Microsoft Technologies and proudly hold the status of a Microsoft Solution Partner for Digital App & Innovation, along with maintaining a Gold Partner legacy of Microsoft in Application Development & Application Integration. We are a dynamic team comprised of dedicated professionals and continue to expand our capabilities and reach.",
```

Comment:
Can you please remove a space from the start point of the quotes from every field?

Reply:
@NishargShah  Done